lib/surface/components/form/text_input.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Surface.Components.Form.TextInput do
+  use Surface.Component
+
+  import Phoenix.HTML.Form
+
+  property form, :string, required: true
+  property field, :string, required: true
+  property value, :string, default: ""
```

Comment:
`value` does not fall under the rules you set from when we define a property. However, it will likely be used more often than other options, and this makes it more ergonomic than passing the keyword. What do you think?

Reply:
Agreed 👍